create_builder.go

```diff
@@ -136,7 +136,7 @@ func (c *Client) createBaseBuilder(ctx context.Context, opts CreateBuilderOption
 		)
 	}
 
-	lifecycle, err := c.fetchLifecycle(ctx, opts.Config.Lifecycle)
+	lifecycle, err := c.fetchLifecycle(ctx, opts.Config.Lifecycle, os)
```

Comment:
Where does `os` come from? Is it pulled from the builder metadata?

Reply:
It comes from the stack build image in the lines above.